lib/omniauth/strategies/strava.rb

```diff
@@ -12,6 +12,13 @@ class Strava < OmniAuth::Strategies::OAuth2
       }
       option :scope, 'read'
 
+      def token_params
```

Comment:
I am not sure why this was required, but it doesnt seem like the `client_id` and the `client_secret` were being added to the request for the token